packages/integration-tests/utils/replayHelpers.ts

```diff
@@ -37,8 +37,10 @@ type SnapshotNode = {
  * @param segmentId the segment_id of the replay event
  * @returns
  */
-export function waitForReplayRequest(page: Page, segmentId?: number): Promise<Request> {
-  return page.waitForRequest(req => {
+export function waitForReplayRequest(page: Page, segmentId?: number): Promise<Response> {
```

Comment:
We actually wait on a response here now, as that may be a bit better/more expected timing.